client/state/action-types.ts

```diff
@@ -865,6 +865,10 @@ export const SITE_FEATURES_FETCH_COMPLETED = 'SITE_FEATURES_FETCH_COMPLETED';
 export const SITE_FEATURES_FETCH_FAILED = 'SITE_FEATURES_FETCH_FAILED';
 export const SITE_FEATURES_REMOVE = 'SITE_FEATURES_REMOVE';
 export const SITE_FRONT_PAGE_UPDATE = 'SITE_FRONT_PAGE_UPDATE';
+export const SITE_INTRO_OFFER_RECEIVE = 'SITE_INTRO_OFFER_RECEIVE';
```

Comment:
I opted for a traditional query/redux pattern over `react-query` because the cart is heavily redux based so this fits better.